src/overlays/actors/ovl_Bg_Dy_Yoseizo/z_bg_dy_yoseizo.c

```diff
@@ -1014,11 +1014,11 @@ void BgDyYoseizo_ParticleDraw(BgDyYoseizo* this, GlobalContext* globalCtx) {
 
     for (i = 0; i < 200; i++, particle++) {
         if (particle->alive == 1) {
```

Comment:
```suggestion
        if (particle->alive == true) {
```
This could probably just be true/false, except there's one spot where you have `particle->alpha = particle->alive = 0;`